Automattic/jetpack
# Publicize: add Fediverse meta tag on posts when available

## Proposed changes:

See https://blog.joinmastodon.org/2024/07/highlighting-journalism-on-mastodon/

If we have an existing Mastodon connection on the site, let's add it as meta tag on each post.

> [!NOTE]
> This first iteration picks up the first available Mastodon connection. I think we should aim to make sure we pick the one that's assigned to the post author. @automattic/jetpack-reach Do you have any ideas on how I could do that?

### Other information:


## Jetpack product discussion

* peLCyI-nZ-p2

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > Settings > Sharing
* Enable Jetpack Social, and connect your site to a test Mastodon account.
* Go to Posts > Add New
* Write a post and publish it. In the post sidebar, ensure the post is shared to Mastodon.
* Check the source of the post on the frontend: you should see this: `<meta name="fediverse:creator" content="@tests@fedi.jeremy.hu" />`
